lib/common.css

```diff
@@ -197,3 +197,13 @@ button.goog-buttonset-default:hover {
   color: <<FORE_SECONDARY>>;
 }
 /** Update End **/
+
+/** Fixes Playlist button **/
+.material .sj-play-button-0 #buttonContent.sj-play-button {
+    background: <<FORE_SECONDARY>>;
+}
+
+.material .sj-play-button-0 #pulse.sj-play-button {
+    background: <<FORE_SECONDARY>>;
+    opacity: .3;
```

Comment:
This is the little pulsing effect around...
Alternatively it can be changed to be ALPHA if you would prefer it:

`background: <<ALPHA 0.3>><<FORE_SECONDARY>><<ALPHA>>`